packages/design-system/src/components/popover.tsx

```diff
@@ -77,10 +77,14 @@ export const PopoverHeader = ({ title }: PopoverHeaderProps) => {
         <PopoverClose asChild>
           <IconButton
             size="2"
-            css={{ marginRight: "$spacing$5" }}
+            css={{
+              marginRight: "$spacing$5",
+              "&:not(:hover):focus": { background: "none" },
```

Comment:
seems like you were trying to achieve this on focus without hover

<img width="81" alt="Screenshot 2022-11-15 at 16 45 51" src="https://user-images.githubusercontent.com/52824/201963099-cb90e6e2-17a6-4ec6-bcb9-1e1fce5ffb29.png">


But in figma it is specified like this:


<img width="166" alt="Screenshot 2022-11-15 at 16 46 45" src="https://user-images.githubusercontent.com/52824/201963275-952ed5d7-0240-49b8-a8ed-8b36d33db03d.png">

Either we need to change it in figma by @taylornowotny and change it on IconButton directly or leave it as-is.

Adhoc changing it in one particular popup is wrong.